packages/block-editor/src/layouts/constrained.js

```diff
@@ -278,3 +294,72 @@ export default {
 		return alignments;
 	},
 };
+
+const POPOVER_PROPS = {
+	placement: 'bottom-start',
+};
+
+function DefaultLayoutJustifyContentControl( {
+	layout,
+	onChange,
+	isToolbar = false,
+} ) {
+	const { justifyContent = 'center' } = layout;
+	const onJustificationChange = ( value ) => {
+		onChange( {
+			...layout,
+			justifyContent: value,
+		} );
+	};
+	const allowedControls = [ 'left', 'center', 'right' ];
+
+	if ( isToolbar ) {
+		return (
+			<JustifyContentControl
+				allowedControls={ allowedControls }
+				value={ justifyContent }
+				onChange={ onJustificationChange }
+				popoverProps={ POPOVER_PROPS }
+			/>
+		);
+	}
+
+	const justificationOptions = [
+		{
+			value: 'left',
+			icon: justifyLeft,
+			label: __( 'Justify items left' ),
+		},
+		{
+			value: 'center',
+			icon: justifyCenter,
+			label: __( 'Justify items center' ),
+		},
+		{
+			value: 'right',
+			icon: justifyRight,
+			label: __( 'Justify items right' ),
+		},
+	];
+
+	return (
+		<ToggleGroupControl
+			__nextHasNoMarginBottom
+			label={ __( 'Justification' ) }
+			value={ justifyContent }
+			onChange={ onJustificationChange }
+			className="block-editor-hooks__flex-layout-justification-controls"
+		>
+			{ justificationOptions.map( ( { value, icon, label } ) => {
+				return (
+					<ToggleGroupControlOptionIcon
+						key={ value }
+						value={ value }
+						icon={ icon }
+						label={ label }
+					/>
+				);
+			} ) }
+		</ToggleGroupControl>
+	);
```

Comment:
I believe this is dead code as `isToolbar` will always be `true`. Can you simplify `DefaultLayoutJustifyContentControl` and remove the `isToolbar` prop?

Reply:
Thank you for pointing this out @noisysocks . I have removed the `isToolbar` props and simplified the component.